public/content/index.html

```diff
@@ -3,16 +3,39 @@
 
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
     <title>Content</title>
 </head>
 
 <body>
-    <a href="./access.html">Search Access</a>
-    <a href="./staff.html">Search Staff</a>
-    <br>
-    <a href="../">Back</a>
-
+    <div class="container">
+        <div class="row">
+            <div class="col">
+                <h1 class="text-center">Cute Kiwi Inc.</h1>
+            </div>
+        </div>
+        <br>
+        <div class="row">
+            <div class="col"></div>
+            <div class="col">
+                <a href="./access.html" role="button" class="btn btn-dark btn-lg btn-block">Search IT data</a>
+            </div>
+            <div class="col">
+                <a href="./staff.html" role="button" class="btn btn-dark btn-lg btn-block">Search Human Resource data</a>
+            </div>
+            <div class="col"></div>
+        </div>
+        <br>
+        <div class="row">
+            <div class="col"></div>
+            <div class="col">
+                <a href="/api/logout" role="button" class="btn btn-secondary btn-lg btn-block">Logout</a>
```

Comment:
Add button to navigate back to the parent.

Reply:
I think, if user wants to see the login/register page, then he/she has to logout first.

Reply:
We are using a file server to deliver web pages. I am not going to modify its logic just to redirect to index.html. It is much easier to add a bunch of buttons.

Reply:
Done.